packages/selinux-policy/fs.cil

```diff
@@ -47,6 +47,7 @@
 (filecon "/.*/usr/bin/storewolf" file api_exec)
```

Comment:
For the lack of a better place to leave this comment. This commit message isn't properly formatted.